app/assets/stylesheets/partials/_sections.scss

```diff
@@ -73,13 +73,37 @@
       }
     }
 
-    &.submit {
+    &.submit, &.analyse {
```

Comment:
Have a look [here](https://pageshot.net/3h43O94NfF6vgg4I/lvh.me). For me it's ok. Contrast on the button is a bit low, but we are not going to use this colour a lot.